### ISSUE TITLE
CreateAssemblyInstanceFromFile unused

### ISSUE THREAD

**Issue by cwollenhaupt** [RickStrahl/wwDotnetBridge#23]
Hi Rick, 

I noticed that the C# code has a method CreateAssemblyInstanceFromFile which calls AppDomain.CreateInstanceAndUnwrap instead of searching all assemblies for the specified type. But it's never called from the wwDotNetBridge.prg file or anywhere in the C# code. 

Is that just unfinished work or did you run into problems with this method?

**Comment by RickStrahl** [comment]
Hmmm... you're right. Just an omission, but you can call this directly for now. I'm adding a function to wwDotnetBridge.prg.

```foxpro
************************************************************************
*  CreateInstanceFromFile
****************************************
***  Function: Creates an instance of an assembly from a file on disk
***            without having to call LoadAssembly first. 
***    Assume: Be careful with this as it doesn't load dependencies.
***            The safer path is to use LoadAssembly then load the type.
***            Mainly useful for loading things like addins out of diff
***            non-application folders.
***      Pass: 
***    Return: object instance on success null or exception on failure
************************************************************************
FUNCTION CreateInstanceFromFile(lcAssemblyFilename, lcTypename, lvParm1, lvParm2, lvParm3)

this.SetError()

loBridge = this.oDotnetBridge
IF ISNULL(loBridge)
   RETURN NULL
ENDIF

lnParmCount = PCOUNT()
DO CASE 
	CASE lnParmCount = 3
		loObject = loBridge.CreateAssemblyInstanceFromFile_OneParm(loInstance,lcAssemblyFilename, lcTypename, lvParm1)	
    CASE lnParmCount = 4
		loObject = loBridge.CreateAssemblyInstanceFromFile_TwoParms(loInstance,lcAssemblyFilename, lcTypename, lvParm1, lvParm2)	
    OTHERWISE
        loObject = loBridge.CreateAssemblyInstanceFromFile_ThreeParms(loInstance,lcAssemblyFilename, lcTypename)
ENDCASE
ENDFUNC
*   CreateInstanceFromFile
```

**Comment by cwollenhaupt** [comment]
Just to provide an update... The problem I actually wanted to solve was that I loaded two DLLs and the second DLL would complain that it cannot load the first one, even though I just successfully loaded it one code line earlier. This would, of course, only happen at runtime, never when debugging my FoxPro application. Turns out that I never considered AssembyLoadContexts because this stuff usually works automatically. Your response helped me figuring this out so I indeed do not need to use this particular method.
